rsc/src/main/scala/rsc/outline/Synthesizer.scala

```diff
@@ -702,6 +683,16 @@ final class Synthesizer private (
     val method = DefnMethod(Mods(Nil), id, Nil, paramss, ret, rhs)
     scheduler(env, method.withPos(tree.pos))
   }
+
+  private def synthesizeTparam(tp: TypeParam): TypeParam = {
+    val mods = Mods(Nil)
+    val id = TptId(tp.id.valueopt.getOrElse("_")).withPos(tp.id.pos)
```

Comment:
Curious: which test case demonstrates the problem that requires changing `.get` to `.getOrElse("_")`?

Also: interesting that his code was duplicated 4 times...